src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/WriteTagCheckTest.java

```diff
@@ -208,7 +208,7 @@ public void testEnumsAndAnnotations() throws Exception {
     public void testNoJavadocs() throws Exception {
         final DefaultConfiguration checkConfig = createModuleConfig(WriteTagCheck.class);
         final String[] expected = {
-            "3: " + getCheckMessage(MSG_MISSING_TAG, "null"),
+            "4: " + getCheckMessage(MSG_MISSING_TAG, "null"),
         };
         verify(checkConfig, getPath("InputWriteTag3.java"), expected);
```

Comment:
same here, rename it please to something related to test name

Reply:
Done